analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -707,6 +707,8 @@ private fun Package.enrichWith(other: Package?): Package =
             description = description.takeUnless { it.isBlank() } ?: other.description,
             authors = authors.takeUnless { it.isEmpty() } ?: other.authors,
             declaredLicenses = declaredLicenses.takeUnless { it.isEmpty() } ?: other.declaredLicenses,
+            declaredLicensesProcessed = declaredLicensesProcessed.takeUnless { declaredLicenses.isEmpty() }
+                ?: other.declaredLicensesProcessed,
```

Comment:
It actually is necessary, because the new changes may alter the processed licenses. These custom changes to the processed licenses would otherwise be lost.

Reply:
But only in a later commit, in this commit the commit message is misleading because it implies that the processed declared licenses from the wrong package could be used which is not the case.

Reply:
It's not that I'm too lazy to adjust the commit message, but I'm simply not sure whether I can follow your argumentation here. IMO the change would have been a sensible one even without the other changes in this PR, as generally you can not assume that the processed license was always created in the default way.